packages/paste-website/src/components/homepage/SiteHeroIllustration.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+import {Box} from '@twilio-paste/box';
+import HomeHero from '../../assets/illustrations/home_hero.svg';
+
+const SiteHeroIllustration: React.FC = () => {
+  return (
+    <Box maxWidth="size70" marginLeft="auto">
+      <HomeHero />
```

Comment:
Is it a SiteHero or a HomeHero? Is it weird the illo is HomeHero but the component is SiteHero?

Reply:
Yeah. Thats strange. HomeHero makes more sense.